canvas_modules/common-canvas/src/common-properties/controls/expression/expression-builder/expression-builder.jsx

```diff
@@ -31,85 +31,86 @@ export default class ExpressionBuilder extends React.Component {
 		this.onChange = this.onChange.bind(this);
 		this.onBlur = this.onBlur.bind(this);
 		this.onSelectionChange = this.onSelectionChange.bind(this);
-		this.lastCursorPos = null;
-
 	}
 
 	onChange(newValue) {
 		const value = (typeof newValue === "string") ? newValue : newValue.toString();
-		let cursor = this.editor.getCursor();
+		const somethingSelected = this.editor.viewState.state.selection.ranges.some((r) => !r.empty);
+		let cursor = this.editor.viewState.state.selection.main.head;
+		if (cursor === 0 && !somethingSelected) { // TODO: Doesn't work when I explicitly set the cursor to 0
```

Comment:
When nothing is selected, cursor is set to the end of the line. But this `if` loop doesn't work when I explicitly set the cursor to line 0, char 0 - 


https://github.com/elyra-ai/canvas/assets/25124000/f70544b1-62eb-4517-a82d-449ae2fcdcbc